modules/openideal_statistics/src/Plugin/Block/OpenidealStatisticsBaseStatisticsBlock.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\openideal_statistics\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Provides base class for openidealEntityStatistics blocks.
+ */
+abstract class OpenidealStatisticsBaseStatisticsBlock extends BlockBase {
+
+  /**
+   * Checks if it's a full view mode.
+   *
+   * @return bool
+   *   True is not provided view mode.
+   */
+  public function isNotViewMode($mode) {
```

Comment:
Wouldn't it better to make it `isViewMode()` or `getViewMode()`?

Reply:
We are checking if the view provided as method argument not active, and return a bool.
For instance, if the method called getViewMode then it should return the actual name of the view mode.